taxi/urls.py

```diff
@@ -6,7 +6,8 @@
     CarDetailView,
     DriverListView,
     DriverDetailView,
-    ManufacturerListView,
+    ManufacturerListView, ManufacturerCreateView, ManufacturerUpdateView, ManufacturerDeleteView, CarCreateView,
+    CarUpdateView, CarDeleteView,
```

Comment:
Group imports using parentheses for better readability as per the checklist. Consider formatting the imports like this: `from .views import (ManufacturerListView, ManufacturerCreateView, ManufacturerUpdateView, ManufacturerDeleteView, CarCreateView, CarUpdateView, CarDeleteView)`.